File: tests/fixtures/codegen/expected/typescript/runtime/queries/admin/events/_Hidden.ts
import type { HiddenParams, HiddenRow } from "../../../../types/queries/admin/events/_Hidden.js";
export type { HiddenParams, HiddenRow } from "../../../../types/queries/admin/events/_Hidden.js";
import type * as pgsid from "./pgsid/queryable.js";
export const hiddenSql = "SELECT 3 AS value;";
export async function hidden(db: pgsid.Queryable): Promise<HiddenRow | undefined> {
    const result = await db.query(hiddenSql, []);
    const row = result.rows[0];
    if (row === undefined)
        return undefined;
    return row as HiddenRow;
}

File: tests/fixtures/codegen/expected/go/queries/admin/events/hidden.go
package events

import "context"

const HiddenSQL = "SELECT 3 AS value;"

type HiddenParams struct {
}
type HiddenRow struct {
	Value int32 `db:"value"`
}

func (q *Queries) Hidden(ctx context.Context) (HiddenRow, error) {
	var row HiddenRow
	err := q.db.QueryRow(ctx, HiddenSQL).Scan(&row.Value)
	return row, err
}
